src/robot.coffee

```diff
@@ -78,7 +91,39 @@ class Robot
   #
   # Returns nothing.
   hear: (regex, options, callback) ->
```

Comment:
This results in `regex` being hidden from view. It should probably be added to the Listener as an option or something...